### ISSUE TITLE
error message issue with hl item add

### ISSUE THREAD

**Issue by misvae** [demonnic/Highlighter#6]
when executing the following command

```
hl item add shamrock#item_important
(Highlighter): could not find category {itemTbl[1]} to add {itemTbl[2]} to
```

I think there's some concatenation issue

**Comment by demonnic** [comment]
looks like I forgot to put the `f` in front of the quotes for string interpolation, will take a look. thanks

**Comment by demonnic** [comment]
fixed locally, will be in the next release

**Comment by demonnic** [comment]
fixed in 1.3.0
